stats/src/test/java/io/airlift/stats/TestTDigest.java

```diff
@@ -279,6 +279,21 @@ public void testUnmergeable()
         }
     }
 
+    @Test
+    public void testTwoValueTDigest()
+    {
+        // TDigest was failing with AssertionError at TDigest:308 with 2 values, while calculating 0.75 quantile. Fixed at TDigest:279
```

Comment:
Let's remove this comment, as it will become stale once the line numbers change. Instead, let's add this explanation to the commit message: "While calculating the 0.75 quantile for a 2-value digest, the flow was falling through the loop triggering the AssertionError"